### DISCUSSION TITLE
Error uploading image: { statusCode: '403', error: 'Unauthorized', message: 'invalid signature' }

### DISCUSSION THREAD

**Discussion by Ronneback** [supabase/supabase#28289] [Questions]
Error uploading image: { statusCode: '403', error: 'Unauthorized', message: 'invalid signature' } 

I have an open storage bucket called "images" in supabase storage with policies INSERT to all public roles. What's wrong here?

Resolved!

**Comment by beverm2391** [comment]
how did you resolve this?

**Comment by cct-upmanyu** [comment]
How did you solve it please explain?
